lib/make/makefile.epa-ng

```diff
@@ -13,22 +13,36 @@ SHELL := /bin/bash
 # Override in your Makefile by setting a parameter *after* the row that
 # includes this file, see documentation in doc/makefile.md.
 #
-__PGM___OPTS =
+EPA_NG_OPTS =
 
 # *** Internal ***
 
-# MAKECALL___PGM__ is a macro that defines what will be output to the .makecall
+# MAKECALL_EPA_NG is a macro that defines what will be output to the .makecall
 # file, the file that records versions, file stamps, parameters etc.
 #
 # *Don't redefine!*
-MAKECALL___PGM___VERSION     = echo "`date +"%Y%m%d %H:%M:%S"`: $@ was made with `__PROGRAM__ --version`" > $@.makecall
-MAKECALL___PGM___PARAMS      = echo "	Called with parameters: $(__PGM___OPTS)" >> $@.makecall
-MAKECALL___PGM___INFILES     = echo "	Input files: $^ (`ls -lL $^|tr '\n' ','`)" >> $@.makecall
-MAKECALL___PGM__             = $(MAKECALL___PGM___VERSION); $(MAKECALL___PGM___PARAMS); $(MAKECALL___PGM___INFILES)
+MAKECALL_EPA_NG_VERSION     = echo "`date +"%Y%m%d %H:%M:%S"`: $@ was made with `epa-ng --version | grep EPA-ng`" > $@.makecall
+MAKECALL_EPA_NG_PARAMS      = echo "    Called with parameters: $(__PGM___OPTS)" >> $@.makecall
+MAKECALL_EPA_NG_INFILES     = echo "    Input files: $^ (`ls -lL $^|tr '\n' ','`)" >> $@.makecall
+MAKECALL_EPA_NG             = $(MAKECALL_EPA_NG_VERSION); $(MAKECALL_EPA_NG_PARAMS); $(MAKECALL_EPA_NG_INFILES)
 
 # *** Targets ***
 
-%.suffix0: %.suffix1 %.suffix1 %.suffixn
-	@$(MAKECALL___PGM__)
-	__PGM_CALL__
-	@echo "`date +"%Y%m%d %H:%M:%S"`: DONE" >> $@.makecall
+# Target to rename the Reference Alignment, if it has been reduced by RAxML to match the epa_result target
+%.raxml.alnfaa: %.raxml.reduced.alnfaa
+        mv $(basename $(basename $@)).raxml.reduced.alnfaa $@
```

Comment:
Okay yes I get that. I will remove this target. I was just thinking of a nice way to not have name conflicts because of the reduced files and also because there are 2 .alnfaa files as input.